alacritty.yml

```diff
@@ -592,8 +592,8 @@
 # - Vi mode exclusive actions:
 #
 #   - Open
-#       Open URLs at the cursor location with the launcher configured in
-#       `url.launcher`.
+#       Performs the action of the first matching mouse enabled hint under the
+#       cursor, if any.
```

Comment:
I don't think the `if any` at the end contributes to this and I'm not entirely certain that `mouse enabled` should be used as a verb, since people might not be aware of this setting. These two issues are minor though.

What I'd definitely like to point out is that we're talking about the vi mode cursor here, not the mouse or terminal cursor. Especially if we keep it at `mouse.enabled` that could be confusing.

Reply:
Pushed up a rewording.